server/routerlicious/packages/services-shared/src/socketIoServer.ts

```diff
@@ -147,6 +147,10 @@ export function create(
 		enableReadyCheck: true,
 		maxRetriesPerRequest: redisConfig.maxRetriesPerRequest,
 		enableOfflineQueue: redisConfig.enableOfflineQueue,
+		retryStrategy(times) {
+			const delay = Math.min(times * 50, 2000);
+			return delay;
+		},
```

Comment:
It seems like this function is the same everywhere. Can we share this somewhere? Like

```ts
export function getRedisClusterRetryStrategy(options: {delayPerAttemptMs: number; maxDelayMs: number}) =>
  (attempts: number) =>
    Math.min(attempts * options.delayPerAttemptMs, options.maxDelayMs);
```

Reply:
Changed